cmd/mt-store-cat/main.go

```diff
@@ -186,7 +188,7 @@ func main() {
 		fmt.Println("Retrieves timeseries data from the cassandra store. Either raw or with minimal processing")
 		fmt.Println()
 		fmt.Println("Usage:")
-		fmt.Printf("	mt-store-cat [flags] <normal|summary> id <metric-id>\n")
+		fmt.Printf("	mt-store-cat [flags] <normal|summary> id <metric-id> <ttl>\n")
```

Comment:
if we accept patterns like the ones in `agg-settings`, isn't it confusing if then we basically ignore every element of the pattern except the `ttl`?